src/omg-cli/omg/tiles/ert/configuration.go

```diff
@@ -155,19 +141,32 @@ func (*Tile) Configure(envConfig *config.EnvConfig, cfg *config.Config, om *ops_
 	}
 
 	properties := Properties{
-		AppsDomain:                 tiles.Value{cfg.AppsDomain},
-		SysDomain:                  tiles.Value{cfg.SysDomain},
-		NetworkingPointOfEntry:     tiles.Value{"external_non_ssl"},
-		ContainerDNSServers:        tiles.Value{"8.8.8.8,8.8.4.4"},
-		SkipSSLVerification:        tiles.BooleanValue{true},
-		HAProxyForwardTLS:          tiles.Value{"disable"},
-		IngressCertificates:        tiles.CertificateValue{tiles.Certificate{cfg.SslCertificate, cfg.SslPrivateKey}},
+		AppsDomain: tiles.Value{cfg.AppsDomain},
+		SysDomain:  tiles.Value{cfg.SysDomain},
+		// NetworkingPointOfEntry:     tiles.Value{"external_non_ssl"},
+		// ContainerDNSServers:        tiles.Value{"8.8.8.8,8.8.4.4"},
+		SkipSSLVerification: tiles.BooleanValue{true},
+		HAProxyForwardTLS:   tiles.Value{"disable"},
+		IngressCertificates: tiles.CertificateValue{[]tiles.CertificateConstruct{
+			{Certificate: tiles.Certificate{cfg.SslCertificate, cfg.SslPrivateKey},
+				Name: "Certificate",
+			},
+		},
+		},
+		CredhubEncryptionKey: tiles.EncryptionKeyValue{[]tiles.EncryptionKey{
+			{
+				Name:    "test",
+				Key:     tiles.KeyStruct{Secret: "aaaaaaaaaaaaaaaaaaaa"},
```

Comment:
Is this necessary to install? If it is we need to generate a proper secret and pipe it through here. An example is how we generate `ops_manager_password` /  `OpsManagerPassword`

Reply:
It is necessary for PCF 2.0. Will do with generated password